nss-day-cohort-47/tabloid-mvc-cocoapebbles
# cc-ticket23

# Description
This includes register functionality to create a new user, I am pushing this up so that everyone can have the base code and the rest of the updated files.
Fixes # (issue)
## Type of change
# Testing Instructions
To test launch TabloidMVC and make sure that you are logged out. From there click the register button which will only appear to guests and unlogged users. Fill out the form and upon completion log in with your new user's email
# Checklist:
